Parse/src/test/java/com/parse/ParseClientConfigurationTest.java

```diff
@@ -105,6 +120,72 @@ public void testSetNetworkInterceptors() {
     assertTrue(collectionsEqual(configurationB.interceptors, collectionB));
   }
 
+  @Test
+  public void testConfigureFromManifest() throws Exception {
+    Context context = mock(Context.class);
+    PackageManager packageManager = mock(PackageManager.class);
+    ApplicationInfo appInfo = mock(ApplicationInfo.class);
+    Bundle metaData = setupMockMetaData(context, packageManager, appInfo);
+    when(metaData.getString(PARSE_SERVER_URL)).thenReturn(serverUrl);
+    when(metaData.getString(PARSE_APPLICATION_ID)).thenReturn(appId);
+    when(metaData.getString(PARSE_CLIENT_KEY)).thenReturn(clientKey);
+
+    Parse.Configuration.Builder builder = new Parse.Configuration.Builder(context);
+    Parse.Configuration config = builder.build();
+    assertEquals(serverUrl + "/", config.server);
+    assertEquals(appId, config.applicationId);
+    assertEquals(clientKey, config.clientKey);
+
+    verify(context).getApplicationContext();
+    verify(context).getPackageManager();
+    verify(context).getPackageName();
+    verify(packageManager).getApplicationInfo(packageName, PackageManager.GET_META_DATA);
+    verify(metaData).getString(PARSE_SERVER_URL);
+    verify(metaData).getString(PARSE_APPLICATION_ID);
+    verify(metaData).getString(PARSE_CLIENT_KEY);
+  }
+
+  @Test(expected = RuntimeException.class)
+  public void testConfigureFromManifestWithoutServer() throws Exception {
+    Context context = mock(Context.class);
+    PackageManager packageManager = mock(PackageManager.class);
```

Comment:
RuntimeEnvironment.getRobolectricPackageManager() is [deprecated](http://robolectric.org/migrating/) in `Robolectric 3.3`
using `shadowOf(packageManager)` instead
Thanks